src/drivers/blockbook.ts

```diff
@@ -0,0 +1,89 @@
+
+import { Buffer } from 'buffer';
+
+import { Address, Coin, Network, UnspentOutput } from '../blockchain';
+import * as driver from '../driver';
+
+export class BlockBookDriver implements driver.Driver {
+  private readonly url: string;
+
+  constructor(public readonly network: Network, prefix?: string) {
+    this.url = `https://tblockbook.peercoin.net`;
```

Comment:
This url is for the testnet blockbook. You must make sure that you support future mainnet deployment as well.

Reply:
You can see how it was done here for the explorer driver: https://github.com/peercoin/perpera/blob/master/src/drivers/explorer.ts#L10